pkg/security/module/cws.go

```diff
@@ -208,6 +202,17 @@ func (c *CWSConsumer) Start() error {
 		}
 	}
 
+	// directory policy provider
+	if provider, err := rules.NewPoliciesDirProvider(c.config.PoliciesDir, c.config.WatchPoliciesDir); err != nil {
+		seclog.Errorf("failed to load policies: %s", err)
+	} else {
+		policyProviders = append(policyProviders, provider)
+	}
+
+	if len(policyProviders) > 1 && policyProviders[0].Type() != rconfig.PolicyProviderType {
+		seclog.Warnf("failed to load remote config policies first: local policies will override remote policies if there are conflicts")
```

Comment:
I would rather make the error message line 199 clearer from a user perceptive. Here, if RC is disable if will fire a error for an incorrect reason ?

Reply:
Can we have multiple directory providers? I put this here as an enforcement mechanism to make sure RC is appended before the directory provider, since all we have is a comment. I could move the enforcement mechanism to a test instead.